### PR TITLE
Update translations for -  Japanese, Azeri and Kyrgyz

### LINKED ISSUE #3722: Cookie/Privacy banner translations for services
**Is your feature request related to a problem? Please describe.**
We have updated translations for these services:

Afaan Oromoo
Afrique
Amharic
Arabic
Azeri
Gahuza
Hausa
Igbo
Indonesian
Japanese
Korean
Pashto
Persian
Pidgin
Punjabi
Somali
Swahili
Thai
Tigrinya
Urdu
Vietnamese
Yoruba

We should update Simorgh so that it has all of the Cookie and Privacy banner translations. Note, these translations will be added together with those in https://github.com/bbc/simorgh/issues/3721, in the issues referenced below

**Describe the solution you'd like**
- Update the service configs at `src/app/lib/config/services/:service.js`
- Check the Simorgh pages load correctly: localhost urls are here: https://github.com/bbc/simorgh/wiki/Simorgh-Pages 

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Before putting the PR through code review, the full cypress test suite should be run (all page types for all services) with `CYPRESS_SMOKE=false npm run test:e2e`

- [ ] This feature is expected to need manual testing.

**Additional context**
N/A

### LINKED ISSUE #3812: Update translations for - Azeri, Japanese, and Kyrgyz
Partly completes: #3721

**Is your feature request related to a problem? Please describe.**
We have updated translations for these services:
Azeri, Japanese, and Kyrgyz.

**Describe the solution you'd like**

- Update the service configs at `src/app/lib/config/services/:service.js`
- Check the Simorgh pages load correctly: localhost urls are here: https://github.com/bbc/simorgh/wiki/Simorgh-Pages

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Before putting the PR through code review, the full cypress test suite should be run (all page types for all services) with CYPRESS_SMOKE=false npm run test:e2e

This feature is expected to need manual testing.

**Additional context**
N/A

### LINKED ISSUE #3834: Update BBC Japanese navigation
**Is your feature request related to a problem? Please describe.**
Update BBC Japanese navigation to include the Rugby World Cup and some other new links

1 Home
    Text Link: ホーム

2 Rugby World Cup
   Text link: ラグビーW杯
    URL:  https://www.bbc.com/japanese/topics/ef55ff8e-8d36-4473-9f11-3ba9fa9ac787

3 Japan
   Text link: 日本
    URL: https://www.bbc.com/japanese/topics/3f53c272-5b8f-4a4f-99de-a857d6726c5b  

4 Asia
   Text link: アジア  
    URL: https://www.bbc.com/japanese/topics/ba90754a-9033-4e9c-990b-d1139e5070a3

5 UK
   Text link: イギリス 
    URL: https://www.bbc.com/japanese/topics/2e91364c-5c77-4660-b76e-d76202785e64

6 US
   Text link: アメリカ  
    URL: https://www.bbc.com/japanese/topics/82857f8e-8134-462a-bb32-b7b14f4eab75

7 Features and Analysis 
   Text link: 解説・読み物
    URL: https://www.bbc.com/japanese/features_and_analysis

8 Video
   Text link: ビデオ
    URL: https://www.bbc.com/japanese/video 

9 World News TV
   Text link: ワールドニュースTV
   URL: https://www.bbcworldnews-japan.com/

 

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by Bopchy**
Partly resolves #3722 
Resolves #3812
Resolves #3834 

**Overall change:** _Update translations for Japanese, Azeri and Kyrgyz._

**Code changes:**

- _Update `src/app/lib/config/services/<service>.js` for Japanese, Azeri and Kyrgyz services._
- _Update footer links._
- _Update Japanese service navigation links._
---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

**Comment by Bopchy** [comment]
Not sure about the spacings and full-stops on `src/app/lib/config/services/japanese.js`

File: src/app/lib/config/services/japanese.js [hunk 1]
```diff
@@ -37,37 +37,35 @@ const service = {
...
           callToActionLinkText: 'BBCニュース　ホームページ',
-          callToActionLast: '',
+          callToActionLast: ' をご覧ください',
           callToActionLinkUrl: 'https://www.bbc.com/japanese',
         },
         500: {
           statusCode: '500',
-          title: '500 - エラー',
+          title: '500 - 内部サーバーエラー',
           message: 'エラーが起きました。ページを読み込み直してください',
```
Comment (AlistairGempf) [inline src/app/lib/config/services/japanese.js]: I think this should be "申し訳ありません。お探しのページは現在開くことができません。こちらをお試しください:"

**Review by DenisHdz — CHANGES_REQUESTED** [review]
There are some translations missing for `Kyrgyz` like `articleTimestampPrefix`, `imageCaptionOffscreenText`, `videoCaptionOffscreenText`, `audioCaptionOffscreenText`, `See all`, `Related content`, `Most read`, etc.

File: src/app/lib/config/services/kyrgyz.js [hunk 2]
```diff
@@ -16,12 +16,12 @@ const service = {
     defaultImage: 'https://news.files.bbci.co.uk/ws/img/logos/og/kyrgyz.png',
     defaultImageAltText: 'BBC News Кыргыз КызMATы',
     dir: `ltr`,
-    externalLinkText: ', external',
+    externalLinkText: ', Би-Би-Сиден тышкары баракча',
     imageCaptionOffscreenText: 'Image caption, ',
```
Comment (AlistairGempf) [inline src/app/lib/config/services/kyrgyz.js]: There are translations for these on the spreadsheet.

Comment (Bopchy) [reply src/app/lib/config/services/kyrgyz.js]: Updated the translations that weren't updated before.

**Comment by Bopchy** [comment]
Kyrgyz translations should be up-to-date with the spreadsheet now

File: src/app/lib/config/services/azeri.js [hunk 3]
```diff
@@ -139,19 +139,24 @@ const service = {
...
+          href:
+            'https://www.bbc.co.uk/privacy/cookies/managing/cookie-settings.html',
           text: 'Cookies',
         },
         {
-          href: 'https://www.bbc.com/contact/',
+          href: 'bbc.com/azeri/institutional-37131054',
           text: 'BBC ilə Əlaqə',
         },
       ],
```
Comment (AlistairGempf) [inline src/app/lib/config/services/azeri.js:166]: This is missing the About the BBC link

Comment (AlistairGempf) [reply src/app/lib/config/services/azeri.js:166]: Whoops, no it isn't! Sorry.

File: src/app/lib/config/services/kyrgyz.js [hunk 4]
```diff
@@ -36,15 +36,15 @@ const service = {
...
-      relatedContent: 'Related content',
+      relatedContent: 'Тема боюнча башка макалалар',
       error: {
         404: {
           statusCode: '404',
-          title: '404 - Барак табылган жок',
+          title: 'Баракча табылган жок',
           message:
             'Бул веб-даректи туура эмес тергенден келип чыгышы мүмкүн. Даректи жана анын жазылышын текшерип көрүңүз.',
           solutions: [
```
Comment (AlistairGempf) [inline src/app/lib/config/services/kyrgyz.js:54]: These have translations available now.

File: src/app/lib/config/services/kyrgyz.js [hunk 5]
```diff
@@ -59,7 +59,7 @@ const service = {
         },
         500: {
           statusCode: '500',
-          title: '500 - Ката',
+          title: 'Ката',
```
Comment (AlistairGempf) [inline src/app/lib/config/services/kyrgyz.js]: The whole 500 error translation seems to have been updated.

File: src/app/lib/config/services/kyrgyz.js (hunk 4 above)
Comment (Bopchy) [reply src/app/lib/config/services/kyrgyz.js:54]: I have added the errors' solutions translations

File: src/app/lib/config/services/japanese.js [hunk 6]
```diff
@@ -132,37 +131,41 @@ const service = {
...
-          href: 'https://www.bbc.com/usingthebbc/cookies/',
-          text: 'Cookies',
+          href: 'https://www.bbc.co.uk/usingthebbc/cookies/',
+          text: 'クッキー',
         },
         {
-          href: 'https://www.bbc.com/contact/',
-          text: 'Contact the BBC',
+          href: 'https://www.bbc.co.uk/contact',
+          text: 'BBCに連絡する',
```
Comment (Bopchy) [inline src/app/lib/config/services/japanese.js:165]: This is now `Contact the BBC` on the Footer tab in the spreadsheet, but it had a translation (the one here) before. I'm not sure if it should be `Contact the BBC`

**Review by AlistairGempf — COMMENTED** [review]
I think the Kyrgyz 404 and 500 messages have been updated. Other than that it all looks good.

**Review by DenisHdz — COMMENTED** [review]
I believe the Azeri `404` and `500` codes need to be removed from the `error title` and the call to action translations should be:

```
callToActionFirst: 'Əks halda, lütfən, ',
callToActionLinkText: 'BBC News Azərbaycanca baş səhifəsinə',
callToActionLast: ' keçin',
```

**Review by DenisHdz — APPROVED** [review]
LGTM, great work 👍

**Review by AlistairGempf — APPROVED** [review]
Good job!

**Review by dr3 — APPROVED** [review]

**Comment by staylos92** [comment]
The linked issue specifies the nav items for Japanese, it doesn't mention keeping the BBC News link as the last item in the nav but we have left it in as part of this PR. Is that correct?

**Comment by staylos92** [comment]
@Bopchy Could you pleas eupdate the Japanese nav to remove the "BBC News" link from the end. Thanks

**Comment by staylos92** [comment]
LGTM 👍

**Pull request merged**